source/faq.blade.php

```diff
@@ -0,0 +1,57 @@
+@extends('_layouts.main')
+
+@section('body')
+  <section class="ud-hero" id="home">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="ud-faq">
+    <div class="shape">
+      <img src="{{ $page->baseUrl }}assets/images/faq/shape.svg" alt="shape" />
```

Comment:
Is this a new image? I didn't found at this PR but the path of svg  is /fag/. Did we need to have a new shape svg file?